<?xml version="1.0"?>
<launch>
  

 
  
  
  <!-- Move base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find navigation)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find navigation)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find navigation)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find navigation)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find navigation)/config/base_local_planner_params.yaml" command="load" />
    
    <remap from="cmd_vel" to="cmd_vel"/>
    <param name="move_base/DWAPlannerROS/yaw_goal_tolerance" value="1.0"/>
    <param name="move_base/DWAPlannerROS/xy_goal_tolerance" value="1.0"/>



  </node>
 
  <!--node pkg="navigation" type="diff_tf.py" name="wheel_odom">
    <rosparam file="$(find navigation)/config/bot_info.yaml" command="load" />
  </node-->

  <node pkg="navigation" type="fake_odom.py" name="fake_wheel_odom"/>

  <node pkg="navigation" type="twist_to_motor.py" name="twist_to_motor">
    <rosparam file="$(find navigation)/config/bot_info.yaml" command="load" />
  </node>
	
	

</launch>


